mozecode/dose
# Gm ctrl

I made the following changes:

- add ontime and moment for stretch goals,
- add getUserDetails function in userCtrl, 
- add other controllers, 
- add user_detail pug
- add paths in welcome page to get to various views
- add connections from welcome to my account summary and back
- change my account view to include other user details
- commenting code
- add script_list view and prescriptionCtrl getAllUserScripts function to produce prescriptions sorted by drug with times and doses included
- add renderCreateScriptForm function and createScript.pug template and test
- add link back to main menu in templates
- add post prescription function and test
- add renderCreateAllergy page and postAllergy functions and test
- minor tweak to allergyCtrl
- get messages working in allergy page to ask user if he/she would like to add another

To test:  
- run ```npm start``` and go to localhost:4000
- log in and test each of the links referenced above.  
- You should be able to register/log in, add a prescription, see a list of prescriptions, and add an allergy
